app/views/static_pages/contact.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>StaticPages#contact</h1>
-<p>Find me in app/views/static_pages/contact.html.erb</p>
+<div class="content-wrap">
+  <div class="container footer-margin">
+    <div class="row">
+      <div class="col-sm-offset-2 col-sm-8 card content-top-margin">
+        <div class="row tutorial-min">
+          <div class="col-sm-12">
+            <h2>Contact Us</h2>
+            <hr />
```

Comment:
Self-closing tags is an XML thing and not necessary in HTML5.  However, if this is a purposeful stylistic decision, self-close your `<br>`'s in the footer: `<br />